_posts/2018-09-05-cockpit-177.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Cockpit 177
+author: pitti
+date: 2018-09-05
+tags: cockpit linux
+slug: cockpit-177
+category: release
+summary: Cockpit with LUKSv2 support and FreeIPA ssh known hosts integration
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly. Here are the release notes from version 177.
+
+### Storage: Support LUKS v2
+
+Cockpit's Storage page can now create, lock, unlock, resize, and otherwise
+configure encrypted devices using the
+[LUKS v2](https://www.saout.de/pipermail/dm-crypt/2017-December/005771.html)
+format.  This offers more flexible unlocking policies, stronger cryptography,
+and better compatibility with future changes and features.
+
+This includes support for LUKS v2 based Network Bound Disk Encryption (NBDE)
+through Clevis and Tang.
+
+### Support centrally managed known ssh hosts
+
+[FreeIPA](https://www.freeipa.org/) centrally manages the list and fingerprints
+of known SSH hosts.  Cockpit's functionality to connect to remote hosts now
+queries this, which avoids asking for fingerprint verification of hosts that
+are already known. This matches the behaviour of the `ssh` command line client.
```

Comment:
> [FreeIPA](https://www.freeipa.org/) centrally manages the list and fingerprints
of known SSH hosts. Cockpit now queries FreeIPA when to avoid fingerprint verification prompts of already-known hosts. This new flow now matches the behavior of the `ssh` command line client.